training-manager.lic

```diff
@@ -80,10 +80,16 @@ class TrainingManager
 
   def check_repair
     return false if @skip_repair
-    return false if Time.now - @repair_timer_snap < @repair_timer
+    
+    if Time.now - @repair_timer_snap > @repair_timer
+      @repair_timer_snap = Time.now
+      return true
+    elsif @repair_every_counter+=1 >= @repair_every
```

Comment:
The `els` is unnecessary (i.e. this can just be an `if`) if that would let you format the code to be clearer.